ivy/data_classes/array/creation.py

```diff
@@ -5,13 +5,15 @@
 
 # local
 import ivy
+from ivy import handle_view
 
 
 # Array API Standard #
 # -------------------#
 
 
 class _ArrayWithCreation(abc.ABC):
+    @handle_view
```

Comment:
@vedpatwardhan Is `ivy.asarray` an edge case? why have a `copy` argument if there is no `view` wrapper? Is `copy` handled somewhere else?

Reply:
@Madjid-CH actually please remove the changes from `creation` and `general` files. These are special cases. After that we are good to merge.